sdk/entra/functions-authentication-events/test/internal/constants.ts

```diff
@@ -0,0 +1,85 @@
+import { TokenClaim } from "@azure/functions-authentication-events";
+
+export class RequestConstants {
+  public static readonly statusMessage = "Ready";
+  public static qsKey1 = "param1";
+  public static qsKey2 = "param2";
+  public static qsValue1 = "value1";
+  public static qsValue2 = "value2";
+  public static readonly source = "/tenants/{tenantId}/applications/{resourceAppId}";
+  public static readonly oDataType = "microsoft.graph.onTokenIssuanceStartCalloutData";
+  public static readonly type = "microsoft.graph.authenticationEvent.TokenIssuanceStart";
+}
+
+export class ResponseConstants {
```

Comment:
Discussed offline, can we have these as plain objects instead of classes?

```ts
export const  RequestConstants = {
   statusMessage : "Ready",
   qsKey1: "param1"
   // etc..
}
```

Reply:
Done